content/user/installing/pick_hosting.md

```diff
@@ -12,5 +12,12 @@ As noted in [Basics - Hosting and Domain concepts](/user/first_steps/hosting/#ho
 their [Recommended Services](https://www.zen-cart.com/content.php?3-services) page.
 
 Further Considerations: 
-- One way to evalute hosts is to run the [test suite offered SSLLabs](https://ssllabs.com/ssltest).  You will want to make sure any host you consider has at least a B rating.  This will help make any PCI scan more likely to succeed.  The more secure your host, the more secure your site.
+
+- One way to evaluate hosts is to run the [test suite offered by SSLLabs](https://ssllabs.com/ssltest).  You will want to make sure any host you consider has an A or A+ rating.  (A B rating might be ok if it's something that can be easily fixed by them, but exercise caution.)  
```

Comment:
"test suite offered". Perhaps remove the word "suite"?

Reply:
It's not just a single test - there are a ton of checks done.

Reply:
True. My rationale was: when I see the word "suite" I assume that "I" have several things to choose amongst. In this case, *they* do all their tests. I have only one option: run their tests.

Reply:
OK.